src/hoverifier.ts

```diff
@@ -453,31 +473,43 @@ export function createHoverifier<C extends object, D, A>({
     )
 
     /** Emits DOM elements at new positions found in the URL */
-    const jumpTargets = allPositionJumps.pipe(
-        // Only use line and character for comparison
-        map(({ position: { line, character, part }, ...rest }) => ({ position: { line, character, part }, ...rest })),
-        // Ignore same values
-        // It's important to do this before filtering otherwise navigating from
-        // a position, to a line-only position, back to the first position would get ignored
-        distinctUntilChanged((a, b) => isEqual(a, b)),
-        map(({ position, codeView, dom, ...rest }) => {
-            let cell: HTMLElement | null
-            let target: HTMLElement | undefined
-            let part: DiffPart | undefined
-            if (isPosition(position)) {
-                cell = dom.getCodeElementFromLineNumber(codeView, position.line, position.part)
-                if (cell) {
-                    target = findElementWithOffset(cell, position.character)
-                    if (target) {
-                        part = dom.getDiffCodePart && dom.getDiffCodePart(target)
-                    } else {
-                        console.warn('Could not find target for position in file', position)
-                    }
-                }
-            }
-            return { ...rest, eventType: 'jump' as 'jump', target, position: { ...position, part }, codeView, dom }
-        })
-    )
+    const jumpTargets = pinningEnabled
```

Comment:
Adapt the docstring to explain why this is only relevant when pinning is enabled.